sample-runner/limestone/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "limestone-runner",
+  "version": "1.0.0",
+  "description": "A tool to run code samples with Enact imports",
+  "author": "Roy Sutton <roy.sutton@lge.com>",
+  "main": "src/index.js",
+  "scripts": {
+    "serve": "enact serve",
+    "pack": "enact pack",
+    "pack-p": "enact pack -p",
+    "watch": "enact pack --watch",
+    "clean": "enact clean",
+    "lint": "enact lint .",
+    "license": "enact license",
+    "test": "enact test start --single-run",
+    "test-watch": "enact test start"
+  },
+  "license": "Apache-2.0",
+  "private": true,
+  "repository": "",
+  "enact": {
+    "theme": "limestone"
+  },
+  "eslintConfig": {
+    "extends": "enact-proxy"
+  },
+  "eslintIgnore": [
+    "node_modules/*",
+    "build/*",
+    "dist/*"
+  ],
+  "dependencies": {
+    "@enact/core": "^4.9.4",
```

Comment:
does limestone support enact 4.9.x ?

Reply:
OK, I copied sandstone but I didn't modified the latest enact yet.
I just modified.